pb.go

```diff
@@ -74,6 +74,7 @@ type ProgressBar struct {
 	ShowPercent, ShowCounters        bool
 	ShowSpeed, ShowTimeLeft, ShowBar bool
 	ShowFinalTime, ShowElapsedTime   bool
+	HideCountersTotal                bool
```

Comment:
I opt to use `Hide` instead of `Show` here to keep compatibility with older version, so existing client won't suddenly have an empty total value & need to opt-in